contracts/Governance.sol

```diff
@@ -88,12 +88,13 @@ contract Governance is TellorVars{
         address _oracle = IController(TELLOR_ADDRESS).addresses(_ORACLE_CONTRACT);
         require(IOracle(_oracle).getBlockNumberByTimestamp(_requestId, _timestamp) != 0, "Mined block is 0");
         address _reporter = IOracle(_oracle).getReporterByTimestamp(_requestId,_timestamp);
-        bytes32 _hash = keccak256(abi.encodePacked(_requestId, _timestamp));
+        bytes32 _hash = keccak256(abi.encodePacked(_requestId, _timestamp, _reporter));
         voteCount++;
         uint256 _id = voteCount;
         voteRounds[_hash].push(_id);
         if (voteRounds[_hash].length > 1) {
             uint256 _prevId = voteRounds[_hash][voteRounds[_hash].length - 2];
+            require(block.timestamp - voteInfo[_prevId].startDate > 12 hours, "this dispute is still open");

```

Comment:
I don't think this is needed.  You can't tally the vote if it's under the vote time right? So the next require statement should grab it

Reply:
Agreed thanks for pointing this out